apps/passport-client/components/screens/ProveScreen/GenericProveScreen.tsx

```diff
@@ -83,29 +87,29 @@ export function GenericProveScreen({
     err(dispatch, "Unsupported request", `Expected a PCD GET request`);
     return null;
   }
-
+  console.log(req);
```

Comment:
remove -- let's avoid having unnecessary `console.log`s when pull request is up for review